client/src/components/MultiJurisdictionAudit/StatusBox/index.tsx

```diff
@@ -64,6 +64,7 @@ const StatusBox: React.FC<IStatusBoxProps> = ({
                 <Button
                   intent="success"
                   loading={isSubmitting}
+                  disabled={isSubmitting}
```

Comment:
`loading` automatically disables the button I believe, so you don't need to add this